packages/sdk/src/types/runtime/actor.ts

```diff
@@ -294,6 +308,35 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
 		return container.context.internal.CreateFromGltf(container, options);
 	}
 
+	/**
+	 * @deprecated
+	 * Use AssetContainer.create*Mesh and [[Actor.Create]] instead.
+	 * @param container The asset container to load the mesh into
+	 * @param options.definition The primitive shape and size
+	 * @param options.addCollider Add an auto-typed collider to the actor
+	 * @param options.actor The initial state of the actor
+	 */
+	public static CreatePrimitive(container: AssetContainer, options: {

```

Comment:
Let's not deprecate this method either. It's very convenient and discoverable for non-power-users.